pkg/cache/cache.go

```diff
@@ -18,7 +18,6 @@ const (
 	GobName = "ocm-cache.gob"
 )
 
-//go:generate mockgen -source=cache.go -package=cache -destination=./cache_mock.go
```

Comment:
Unrelated package's mock service changed

Reply:
Same - now in Makefile.